actions/installer/action.yml

```diff
@@ -20,9 +20,16 @@ branding:
 runs:
   using: 'composite'
   steps:
+    - name: Get Action Ref
```

Comment:
we don't need this step if all it does it use the ACTION_REF. You can reference it directly using `${{ github.action_ref }}`

Reply:
Done